src/components/Users.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react";
+import getAllUsers from "../utils/api";
+
+let userData = {};
+
+const Users = () => {
+  const [userData, setUserData] = useState([]);
```

Comment:
If you are going to use the user elsewhere in your app it may make sense to store this in a context